index.js

```diff
@@ -10,3 +10,5 @@ import { forumCategoriesObj } from "./helpers.js";
 const copyright = document.getElementById("copyright");
 
 copyright.innerText = new Date().getFullYear();
+const postsContainer = document.getElementById("posts-container");
+console.log(postsContainer);
```

Comment:
```suggestion
```
Once you delete this line, we can merge.

Reply:
deleted console.log :)